x-pack/plugins/canvas/public/components/hooks/workpad/use_incoming_embeddable.ts

```diff
@@ -45,10 +49,19 @@ export const useIncomingEmbeddable = (selectedPage: CanvasPage) => {
       if (originalElement) {
         const originalAst = fromExpression(originalElement!.expression);
 
-        const functionIndex = originalAst.chain.findIndex(
-          ({ function: fn }) => fn === 'embeddable'
+        const functionIndex = originalAst.chain.findIndex(({ function: fn }) =>
+          ['embeddable', 'savedVisualization'].includes(fn)
         );
 
+        if (functionIndex === -1) {
+          return;
```

Comment:
I suspect we'll want to dispatch `fetchAllRenderables` here as well to prevent the elements stuck in loading.
```suggestion
          dispatch(fetchAllRenderables());
          return;
```